### PR TITLE
Improve Compliance Notification API

### PR DESCRIPTION
**Pull request by chkp-dolevh**

### CODE REVIEW

File: website/docs/d/continuous_compliance_notification.html.markdown [hunk 1]
```diff
@@ -29,8 +29,128 @@ The following arguments are supported:
 In addition to all arguments above, the following attributes are exported:
 
 * `name` - Notification policy name.
-* `description` - Description of the notification.
-* `alerts_console` - Include in the alerts console.
-* `scheduled_report` - Scheduled report details.
-* `change_detection` - Change detection options.
+  The following arguments are supported:
```
Comment (itzikYeret) [inline website/docs/d/continuous_compliance_notification.html.markdown]: this results in the same line as previous line you need a new line

**Review by itzikYeret — CHANGES_REQUESTED** [review]

File: dome9/resource_dome9_continuous_compliance_notification.go [hunk 2]
```diff
@@ -700,3 +858,17 @@ func flattenEmailPerFindingData(respEmailPerFindingData *continuous_compliance_n
 
 	return []interface{}{m}
 }
+
+func ValidatePayloadFormatJSON(v interface{}, k string) (ws []string, errors []error) {
+	// IAM Policy documents need to be valid JSON, and pass legacy parsing
```
Comment (itzikYeret) [inline dome9/resource_dome9_continuous_compliance_notification.go]: remove this line

File: dome9/resource_dome9_continuous_compliance_notification.go [hunk 3]
```diff
@@ -700,3 +858,17 @@ func flattenEmailPerFindingData(respEmailPerFindingData *continuous_compliance_n
 
 	return []interface{}{m}
 }
+
+func ValidatePayloadFormatJSON(v interface{}, k string) (ws []string, errors []error) {
```
Comment (itzikYeret) [inline dome9/resource_dome9_continuous_compliance_notification.go:862]: is this ok for empty json on null values?

File: website/docs/r/continuous_compliance_notification.html.markdown [hunk 4]
```diff
@@ -27,6 +28,8 @@ resource "dome9_continuous_compliance_notification" "test_notification" {
     external_ticket_creating_state     = "EXTERNAL_TICKET_CREATING_STATE"
     aws_security_hub_integration_state = "AWS_SECURITY_HUB_INTEGRATION_STATE"
     webhook_integration_state          = "WEBHOOK_INTEGRATION_STATE"
+    slack_integration_state            = "SLACK_INTEGRATION_STATE"
```
Comment (itzikYeret) [inline website/docs/r/continuous_compliance_notification.html.markdown:31]: please add webhook example

File: dome9/resource_dome9_ruleset_test.go [hunk 5]
```diff
@@ -99,7 +99,6 @@ resource "%s" "%s" {
 	cloud_vendor       = "aws"
 	language           = "en"
 	hide_in_compliance = false
-	is_template        = false
```
Comment (itzikYeret) [inline dome9/resource_dome9_ruleset_test.go:102]: I think you are not align with master, please align

**Review by itzikYeret — CHANGES_REQUESTED** [review]

File: website/docs/r/continuous_compliance_notification.html.markdown (hunk 4 above)
Comment (dolevh-d9) [reply website/docs/r/continuous_compliance_notification.html.markdown:31]: yep

File: dome9/resource_dome9_continuous_compliance_notification.go (hunk 3 above)
Comment (chkp-dolevh) [reply dome9/resource_dome9_continuous_compliance_notification.go:862]: yep

File: dome9/resource_dome9_ruleset_test.go (hunk 5 above)
Comment (chkp-dolevh) [reply dome9/resource_dome9_ruleset_test.go:102]: No, that ok

**Review by itzikYeret — APPROVED** [review]

**Pull request merged**